tests/container

```diff
@@ -155,6 +155,13 @@ lxc stop c1
 lxc start c1
 waitInstanceBooted c1
 
+echo "==> Ensure aliased image won't launch with VM flag set"
+lxc stop c1
+lxc image alias create containeralias "$(lxc config get c1 volatile.base_image)"
+! lxc launch containeralias --vm || false
+lxc start c1
+waitInstanceBooted c1
```

Comment:
I don't think the alias and `lxc launch` bit needs to be done while `c1` is stopped as the base image is going to be untouched anyway.